Sources/KituraContracts/CodableQuery/QueryEncoder.swift

```diff
@@ -405,7 +479,18 @@ public class QueryEncoder: Coder, Encoder, BodyEncoder {
         func encodeNil() throws {}
 
         func encode<T>(_ value: T) throws where T : Encodable {
-            let _: [String : String] = try encoder.encode(value)
+            try encoder._encode(value: value)
         }
     }
 }
+
+infix operator =+=
+
+    func =+= (lhs: inout String?, rhs: String) {
```

Comment:
This needs a comment explaining why it's here.  (used to build a comma-separated list of values for a given `fieldName` when encoding a `[Date]` which uses a custom formatting).